src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIRemoteAPIService.java

```diff
@@ -0,0 +1,120 @@
+package ca.corefacility.bioinformatics.irida.ria.web.services;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import ca.corefacility.bioinformatics.irida.exceptions.IridaOAuthException;
+import ca.corefacility.bioinformatics.irida.model.RemoteAPI;
+import ca.corefacility.bioinformatics.irida.model.RemoteAPIToken;
+import ca.corefacility.bioinformatics.irida.model.project.Project;
+import ca.corefacility.bioinformatics.irida.model.remote.RemoteStatus;
+import ca.corefacility.bioinformatics.irida.ria.web.ajax.CreateRemoteProjectRequest;
+import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.RemoteAPIModel;
+import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.RemoteProjectModel;
+import ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.ajax.AjaxCreateItemSuccessResponse;
+import ca.corefacility.bioinformatics.irida.service.ProjectService;
+import ca.corefacility.bioinformatics.irida.service.RemoteAPIService;
+import ca.corefacility.bioinformatics.irida.service.RemoteAPITokenService;
+import ca.corefacility.bioinformatics.irida.service.remote.ProjectRemoteService;
+
+/**
+ * UI Service to handle request for Remote APIs
+ */
+@Component
+public class UIRemoteAPIService {
```

Comment:
What's the point of this class?  It's just re-wrapping other service methods.  Every method is only called from one place.  Why not just have the functionality in those places?